profiles/SampleCleric-setup.yaml

```diff
@@ -1,129 +1,709 @@
+favor_god: Hav'roth
+immortal_aspect: cobra
+use_favor_altars: false
+favor_goal: 25
+
+#if you have skates for the ice road, put in the noun of the shoes you're taking off. if none, leave blank
+footwear:
+
+#name the worn instrument you intend to use for Performance - typically zills or cowbell
+worn_instrument: zills
+
+#search for classes to listen to in your safe-room
+listen: true
+
+#compendium/textbook for first-aid
+#false for a compendium
+textbook: false
+textbook_type: textbook
+
+#any container that may contain training items
+storage_containers:
+- hip.pouch
+- backpack
+- lootsack
+- satchel
+- thigh.bag
+
+skinning:
+  #skin: true or false
+  skin: true
+  #number of individual arranges - only use when skinning slows down
+  arrange_count: 0
+  #tie a bundle to fit more skins for sale
+  tie_bundle: true
+
+#skills to train while in town
+crossing_training:
+- Theurgy
+- First Aid
+- Athletics
+- Augmentation
+- Utility
+- Warding
+- Outdoorsmanship
+- Engineering
+- Appraisal
+- Attunement
+#######################################################################################################
+#Container all theurgy supplies are stored
+theurgy_supply_container: satchel
+
+#edged object to light incense with flint
+flint_lighter: sword
+
+#container holding holy water
+water_holder: chalice
+
+#god to use with osrel (changes look of OM orb)
+osrel_favor_god: Firulf
+
+#Turn of harness before infuse for OM, requires skill threshold
+osrel_no_harness: true
+
+#Much much to infuse each round
+osrel_amount: 10
+
+#Perform last rites on NON-UNDEAD critters while hnting
+last_rites: true
+
+#Tithe to the local church, raises devotion
+tithe: true
+
+#prayer mat to use while hunting, teaches theurgy, advanced option
+prayer_mat: prayer tapestry
+#######################################################################################################
+#Offensive spells to use in combat
+#Spells are stored in data/base-spells.yaml
+offensive_spells:
+- name: Horn of the Black Unicorn
+  cast_only_to_train: true
+  max_threshold: 2 #Cast if 2 or fewer critters in room
+  mana: 10
+- name: Fire of Ushnish
+  cast_only_to_train: true
+  mana: 30
+  min_threshold: 3 #Cast if 3 or more critters in room
+- name: Malediction
+  cast_only_to_train: true
+  mana: 5
+  cast: cast malediction
+
+#cambrinth settings  
+cambrinth: cambrinth ring
+cambrinth_cap: 5
+#Number of times to charge a cambrinth item with use_auto_mana
+cambrinth_num_charges: 2
+#helpful for traders or any time you have "extra" mana left in a camb
+cambrinth_invoke_exact_amount: true
+#rotates cambrinth charges and harness to train Attunement and Arcana
+use_harness_when_arcana_locked: true
+#all buff_spells to use harness when arcana locked
+combat_trainer_buffs_force_cambrinth: false  
+
+#the % of mana to use with auto_mana.  example .8 would use 8 mana on a discern of 10
+#new chars don't have a .8 possible with mana cost, change this to 1 for early ranks.
```

Comment:
❓ This highlights what I think is the philosophical difference between what these YAMLs used to be and what this PR makes them. I'd always intended them to be starting places, so they used gear, spells, prep, etc. that a new character would be likely to have. Likewise they had been fairly minimal in what settings they had, with the assumption people would learn about more as they go.

Do you think that is no longer necessary?

I'm not saying these changes are right or wrong, it's just a lot to take in with no discussion beforehand. These appear to me to be a pretty dramatic change in direction.

Reply:
> no discussion beforehand

To clarify, I meant "no discussion beforehand with me". I'm just trying to say that I'm lacking some context around these changes. I assume there was a lively discussion in Slack and a recap could help me get up to speed.

Reply:
Just my two cents.  When I made the sample empath file I specifically tried to include QOL options and answer as many of the super frequently asked questions as possible.  It's as much self-defense to avoid answering the same questions dozens of times as it is a template for new characters.  Which is why I also tried to comment on pretty much every option about what/why it is used.  I've been asked several times now to update the non-empath files to match and I think that's what was done with this PR.

Reply:
Well, I think one of the issues is that the system has expanded since those were written. Another issue is that most people return to characters that already have substance and are well past the basics. Maybe we can find a happy medium? I don't actually think most of these are that crazy though, all of this stuff is things I do with my characters from level 1. We can minimize spell lists and remove anchors and such, but apart from that (and maybe other things people can think of), level 1+ uses all of these things.

Reply:
I just think dr-scripts has grown past the "this is for f2p noobs" thing.

Reply:
Maybe we rename them `AdvancedClass-setup.yaml`?

Reply:
I could agree that the vast major of folks I've helped lately are 50 or 100+.  Lich newb not DR newb so they know what behavior they want, just not how to make Lich do the things.

Reply:
To clarify, the things she's included, are things she (and I) answer on a daily basis.

Reply:
> answer as many of the super frequently asked questions as possible
> she (and I) answer on a daily basis.

To me, that's a very compelling reason!